examples/vision/image_enhancer.py

```diff
@@ -0,0 +1,204 @@
+"""
+Title: Image Enhancer/Denoiser
+Author: [Anish B](https://twitter.com/anishhacko)
+Date created: 2021/12/31
+Last modified: 2022/01/05
+Description: Implementation of Simple Image Enhancer/Denoiser for Document Image Enhancements.
+"""
+"""
+## Introduction
+
+This example explains a simple "Image Enhancer/Denoiser" using UNet architecture. The
+problem we focus here is enhancing the document images which are deteriorated through
+external degradations like blur, corrupt text blocks etc... ,we have followed up on few
+seminal papers as reference and used simple pretrained UNet as encoder and Efficient
+Sub-Pixel CNN as decoder. we also do not attempt super-resolution as our primary task hence we
```

Comment:
Can you highlight how this example differs from https://keras.io/examples/vision/super_resolution_sub_pixel/ ?

Reply:
@fchollet, you want me to write over in response or in the introduction block?